<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>642</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Vipster</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/vipster.qrc">
    <normaloff>:/images/vipster.png</normaloff>:/images/vipster.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1241</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVipster">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuFrom_existing_Molecule">
     <property name="title">
      <string>&amp;From existing Molecule</string>
     </property>
     <addaction name="actionCopy_single_Step"/>
     <addaction name="actionCopy_current_Selection"/>
     <addaction name="actionCopy_Trajector"/>
    </widget>
    <widget class="QMenu" name="menuLoad_Parameter_set">
     <property name="title">
      <string>Load &amp;Parameter set</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuLoad_IO_Preset">
     <property name="title">
      <string>Load &amp;IO Preset</string>
     </property>
    </widget>
    <addaction name="actionNew_Molecule"/>
    <addaction name="menuFrom_existing_Molecule"/>
    <addaction name="actionLoad_Molecule"/>
    <addaction name="actionSave_Molecule"/>
    <addaction name="separator"/>
    <addaction name="menuLoad_Parameter_set"/>
    <addaction name="actionSave_Parameter_set"/>
    <addaction name="separator"/>
    <addaction name="menuLoad_IO_Preset"/>
    <addaction name="actionSave_IO_Preset"/>
    <addaction name="separator"/>
    <addaction name="actionScreenshot"/>
    <addaction name="actionScreenshot_all_Steps"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionNew_Atom"/>
    <addaction name="actionDelete_Atom_s"/>
    <addaction name="actionCut_Atom_s"/>
    <addaction name="actionCopy_Atom_s"/>
    <addaction name="actionPaste_Atom_s"/>
    <addaction name="separator"/>
    <addaction name="actionRename_Atom_s"/>
    <addaction name="actionHide_Atom_s"/>
    <addaction name="actionShow_Atom_s"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Vipster"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuVipster"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew_Molecule">
   <property name="text">
    <string>&amp;New Molecule</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionLoad_Molecule">
   <property name="text">
    <string>&amp;Load Molecule</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Molecule">
   <property name="text">
    <string>&amp;Save Molecule</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_Parameter_set">
   <property name="text">
    <string>Sa&amp;ve Parameter set</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>S&amp;creenshot (current Step)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew_Atom">
   <property name="text">
    <string>&amp;New Atom</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionDelete_Atom_s">
   <property name="text">
    <string>&amp;Delete Atom(s)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionCopy_Atom_s">
   <property name="text">
    <string>&amp;Copy Atom(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste_Atom_s">
   <property name="text">
    <string>&amp;Paste Atom(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionHide_Atom_s">
   <property name="text">
    <string>&amp;Hide Atom(s)</string>
   </property>
  </action>
  <action name="actionShow_Atom_s">
   <property name="text">
    <string>&amp;Show Atom(s)</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionAbout_Vipster">
   <property name="text">
    <string>&amp;About Vipster</string>
   </property>
  </action>
  <action name="actionCopy_single_Step">
   <property name="text">
    <string>&amp;Copy single Step</string>
   </property>
  </action>
  <action name="actionCopy_Trajector">
   <property name="text">
    <string>C&amp;opy Trajectory</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionSave_IO_Preset">
   <property name="text">
    <string>Save I&amp;O Preset</string>
   </property>
  </action>
  <action name="actionParamDummy">
   <property name="text">
    <string>&amp;paramDummy</string>
   </property>
  </action>
  <action name="actiondummy">
   <property name="text">
    <string>dummy</string>
   </property>
  </action>
  <action name="actionCut_Atom_s">
   <property name="text">
    <string>C&amp;ut Atom(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy_current_Selection">
   <property name="text">
    <string>Copy current &amp;Selection</string>
   </property>
  </action>
  <action name="actionSet_Bonds">
   <property name="text">
    <string>(Re-)Calculate &amp;Bonds</string>
   </property>
  </action>
  <action name="actionRename_Atom_s">
   <property name="text">
    <string>&amp;Rename Atom(s)</string>
   </property>
  </action>
  <action name="actionScreenshot_all_Steps">
   <property name="text">
    <string>Screenshot (all Steps)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/vipster.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_Vipster</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_Molecule</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newMol()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>571</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_Molecule</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadMol()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_Molecule</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveMol()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_IO_Preset</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>savePreset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_Parameter_set</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveParam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScreenshot</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveScreenshot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuEdit</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>MainWindow</receiver>
   <slot>editAtoms(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScreenshot_all_Steps</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveScreenshots()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuFrom_existing_Molecule</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>MainWindow</receiver>
   <slot>newMol(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>editAtoms(QAction*)</slot>
  <slot>about()</slot>
  <slot>newMol()</slot>
  <slot>stepBut(QAbstractButton*)</slot>
  <slot>loadMol()</slot>
  <slot>saveMol()</slot>
  <slot>savePreset()</slot>
  <slot>saveParam()</slot>
  <slot>saveScreenshot()</slot>
  <slot>newMol(QAction*)</slot>
  <slot>setMult(int)</slot>
  <slot>saveScreenshots()</slot>
 </slots>
</ui>
